principles/fp-002-format.md

```diff
@@ -14,15 +14,17 @@ The ontology is made available in a common formal language in an accepted concre
 
 A common format allows the maximum number of people to access and reuse an ontology.
 
-## Implementation
+## Recommendations and Requirements
+
+All ontologies MUST have at least one OWL product whose name corresponds to the registered id. Thus the ontology whose IRI is http://purl.obolibrary.org/obo/ro.owl (known to the  OBO Foundry as 'RO'), must have at least the product ro.owl. Developers are free to use whatever combination of technologies and formats is appropriate for development. However, the official OWL PURL for the ontology must resolve to a syntactically valid OWL file using the [RDF-XML](https://www.w3.org/TR/rdf-syntax-grammar/) syntax.
 
-### Recommendations and Requirements
+Ontologies can OPTIONALLY produce an OBO-Format file. This is conventionally the same IRI as the owl, but with .owl changed to .obo. Note that an obo product is not listed by default. If you produce an OBO format product, you should register it under the 'products' field in the appropriate metadata file found in this [folder](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology).
```

Comment:
what about OBO Graph JSON?

Reply:
(Just an FYI: The text above was pulled from existing text found within the criteria for review link for this principle.)

I don't know enough about that format. I would say that any format convertible by ROBOT or Protege is fair game. Is that the case for OBO Graph JSON? If so, I'll generalize the statement and add that as another example.

Reply:
Yes, ROBOT also will generate OBO Graph JSON. I think you're right that it's good to mention that any other exports will probably help people

Reply:
My suggestion would be to replace the first sentence of this paragraph with: "Ontologies can OPTIONALLY produce products in other formats. For historical reasons, OBO-Format files are often produced." Then the rest of the paragraph (as written) is an example of a .obo product.